tasks/Create-A-wall.md

```diff
@@ -0,0 +1,27 @@
+#Labyrinth: Create a Wall
+
+The [FOSSASIA Labyrinth](https://github.com/fossasia/labyrinth/) allows you to contribute parts to a huge labyrinth.
+We can add new features to the labyrinth which improve the game.
+
+Requirements:
+- Download and install Inkscape 
+- Create a Wall for the Labyrinth maze 
+- Add your wall to labyrinth/tiles/rooms/wall
+- A merged pull request
+
+Expected outcome:
+- Added atleast one wall
+- One pull requset is succesfully merged
+
+Hints:
+- You can use techniques like [pair programming](https://www.youtube.com/watch?v=vgkahOzFH2Q) or collaborate together on branches of your forks. You can attribute a pull-request also to other people who helped you out or submit a pull-request where you and an other person contributed together.
+- You can join our [chat on gitter](https://gitter.im/fossasia/labyrinth) if you get stuck to get help or just hang out and have a nice time with others.
+
+---
+
+Categories:
+- [X] Code
+- [X] User Interface
+- [ ] Documentation / Training
+- [X] Quality Assurance
```

Comment:
Is this quality assurance in your eyes? If yes, all fine!

Reply:
Yes it is I think